script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_4_1C.txt

```diff
@@ -1,63 +1,63 @@
 <Page1>
 ―――I'm attacked by a headache.
-Small thorns bounce around in my eye sockets.
-A red painful sensation as if it's trying to deny what I just witnessed.
+Needles bounce around in my eye sockets.
+A painful red sensation denies what I just witnessed.
 <Page2>
-"...Come on. I just wasted my energy even more."
+"...Come on. I just wasted even more energy."
 <Page3>
-The girl looks back at me with her shoulders slumped in disappointment.
+The woman looks back at me with her shoulders slumped in disappointment.
 From the start, she had no interest in the corpses that have just vanished.
 <Page4>
-"This is all because of you standing there and hesitating.
-If you had decided sooner, we would never have been found in the first place!"
+"This is all because you just standing there.
+If you had decided sooner, we never would have been found in the first place!"
 <Page5>
-The girl briskly walks up to where I have fallen on my backside.
+The woman briskly walks up to where I have fallen on my backside.
 The black panther has long since disappeared.
 Maybe it saw what she just did and ran away, thinking it was no match for her.
 <Page6>
 "Hey. What was that just now?"
-...A headache forms right next to my eyeballs.
-The pain penetrates my skull and disrupts my linguistic cortex.
-What the hell―――is this girl?
+...A headache forms behind my eyeballs.
+The pain penetrates my skull and disrupts the language center of my brain.
+What the hell―――is this woman?
 <Page7>
 "Hm? You mean that? That was just a corpse, right?"
 "Those are familiars that belong to an enemy vampire. It's a bit unusual... or rather, it's the first time I've seen this type."
 "No, I'm not talking about that. I'm talking about you."
 <Page8>
 The sharp headache urges me to come to my senses.
-This girl is a monster.
+This woman is a monster.
 The black panther that's out of place in a modern city,
 and even the impossibly burning corpses,
-both seem far more believable when compared to this girl.
+both seem far more believable when compared to this woman.
 <Page9>
 Creating a tornado by simply waving her arms? This is ridiculous. As a fragile human being, there's no way I can live with such a thing for even a second.
 <Page10>
 "Me? I already told you that I'm a vampire.
 Wait, are you telling me that you don't know the meaning of the word 'vampire'?
-I thought it would have been common knowledge in Japan."
+I thought it would be common knowledge in Japan."
 <Page11>
-The girl gives an idiotic response.
+The woman gives an idiotic response.
 I take this opportunity to put some strength into my trembling limbs.
-After what just happened, the girl has moved away from the entrance to the alleyway.
+After all that, she moves to the entrance to the alleyway.
```

Comment:
This is wrong. Arc moved **away** from the entrance, creating a gap for Shiki to run.

During all the commotion, she had moved away from the entrance to the alleyway.

Reply:
Yeah this is a regression. Yusuke's comment is a good reword.

Reply:
After reading it again, I noticed that my rewording makes it sounds like she moves towards the alleyway from the entrance. Should probably refer to it as 'alleyway entrance' to avoid confusion.